Makefile

```diff
@@ -113,7 +134,82 @@ preload-images:
 
 validate: pkg/subctl/operator/common/embeddedyamls/yamls.go
 
-.PHONY: test validate build ci clean generate-clientset generate-embeddedyamls generate-operator-api operator-image preload-images
+all: manager
+
+# Run tests
+#test: generate fmt vet manifests
+	#go test ./... -coverprofile cover.out
+
+# Build manager binary
+manager: generate fmt vet
+# We check BUILD_ARGS since that's what the compile script uses
+ifeq (--debug,$(findstring --debug,$(BUILD_ARGS)))
+	go build -o bin/submariner-operator main.go
+else
+	go build -ldflags -s -w -o bin/submariner-operator main.go
+endif
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet manifests
+	go run ./main.go
+
+# Install CRDs into a cluster
+install: manifests
+	$(KUSTOMIZE) build config/crd | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl apply -f -
+
+# Uninstall CRDs from a cluster
+uninstall: manifests
+	$(KUSTOMIZE) build config/crd | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl delete -f -
+
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: manifests clusters preload-images
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl apply -f -
+
```

Comment:
All this is unnecesary in our case, our testing is more complex than that, because we do multicluster, all this is handled from shipyard/subctl. etc.

Reply:
@mangelajo thanks I will remove this part